m10_point_of_interest/03_presentation_maker.rst

```diff
@@ -0,0 +1,61 @@
+Presentation Maker
+==================
+
+What is it?
+-----------
+
+Presentation Maker is a tool for making HTML & PDF presentations from the point of interest summary blocks (POI).
+
+Presentation Maker finds all used POI directives from the reStructuredText (RST) course material and makes a presentation out of them.
+
+More at the `Presentation Maker documentation <https://github.com/Aalto-LeTech/presentation-maker>`_.
+
+Why to use Presentation Maker?
+------------------------------
+
+Purpose of the Presentation Maker is to **eliminate separate process of creating presentation slides.** It all comes down to creating good POI which can be used as a slides.
+
+If you start creating POI summary blocks with the same mindset as creating presentation slides you can just run Presentation Maker and in seconds you have a set of slides ready.
+
+How it works
+------------
+
+In order to use Presentation Maker effectively you will need to know a little bit how it works.
+
+**Each POI represents a slide.**
+
+When you make POI keep in mind that it creates a slide. So make content to look like a slide.
+
+**Presentation Maker systematically searches POI from the start of the course material to till end**
+
+Presentation Maker does not care about links between POI. It just adds POI to the presentation in the order as they appear on the content. You can of course choose what rounds will be included in the presentation.
+
+**Sometimes POI and the slides do not look the same**
```

Comment:
Admonition?

Reply:
I made this as a admonition and moved it under the title "making a good POI". 

I added "How can I edit the last slide?" to this section (how it works).